x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/crawler/automatic_crawl_scheduler/automatic_crawl_scheduler_logic.ts

```diff
@@ -148,11 +175,25 @@ export const AutomaticCrawlSchedulerLogic = kea<
       } else {
         actions.deleteCrawlSchedule();
       }
+      actions.submitConnectorSchedule({
+        ...values.index.connector.scheduling,
+        enabled: values.crawlAutomatically && values.useConnectorSchedule,
+      });
     },
-    setCrawlUnit: actions.saveChanges,
+    setCrawlAutomatically: actions.saveChanges,
     setCrawlFrequency: actions.saveChanges,
+    setCrawlUnit: actions.saveChanges,
     setUseConnectorSchedule: actions.saveChanges,
-    toggleCrawlAutomatically: actions.saveChanges,
+    submitConnectorSchedule: async ({ scheduling }) => {
+      try {
```

Comment:
suggestion: This try catch block isn't going to do anything, because the makeRequest action always succeeds. Should move the flashErrors to listening to the error action instead.